pkg/commands/plugin/capabilities.go

```diff
@@ -0,0 +1,62 @@
+package plugin
+
+import (
+	"github.com/urfave/cli"
+	"github.com/vapor-ware/synse-cli/pkg/client"
+	"github.com/vapor-ware/synse-cli/pkg/formatters"
+	"github.com/vapor-ware/synse-cli/pkg/utils"
+)
+
+const (
+	// capabilitiesCmdName is the name for the 'capabilities' command.
+	capabilitiesCmdName = "capabilities"
+
+	// capabilitiesCmdUsage is the usage text for the 'capabilities' command.
+	capabilitiesCmdUsage = "Get capability information from a plugin"
+
+	// capabilitiesCmdDesc is the description for the 'capabilities' command.
+	capabilitiesCmdDesc = `The capability command gets capability information from a plugin via the
+  Synse gRPC API. The device information returned here is similar to that
+  of a 'synse server capabilities' command, but contains more information
+  about the device.
```

Comment:
I'm not sure its true that it contains more info about the device.

I think the usage should say something like
> Get the device capabilities for a plugin

and the description could say something like
> The capabilities command gets information on the kinds of devices that a plugin supports as well as the types of reading outputs each of those devices support.